lib/rules/font-family-no-missing-generic-family-keyword/index.cjs

```diff
@@ -63,10 +63,10 @@ const rule = (primary, secondaryOptions) => {
 		}
 
 		root.walkDecls(/^font(-family)?$/i, (decl) => {
-			// Ignore @font-face
+			// Ignore @font-face and @font-palette-values
 			const parent = decl.parent;
-
-			if (parent && typeGuards.isAtRule(parent) && parent.name.toLowerCase() === 'font-face') {
+			const rules = ['font-face', 'font-palette-values'];
```

Comment:
`npm run build` should have added the missing newline for you.

Reply:
v23.3.0, but as I said, I can’t really use `npm run format` and `npm run build`, since it’s formatting wrongly. Should I switch node version, does this help?

Reply:
> Should I switch node version, does this help?

I was just curious; 23 is fine.

> …since it’s formatting wrongly

I don't know what's going on, but since there is only one file to edit, what about doing it manually?
i.e.
```suggestion
			const rules = ['font-face', 'font-palette-values'];

```